Hackney.Core/DynamoDb/Converters/DynamoDbDateTimeConverter.cs

```diff
@@ -0,0 +1,31 @@
+using Amazon.DynamoDBv2.DataModel;
+using Amazon.DynamoDBv2.DocumentModel;
+using System;
+
+namespace Hackney.Core.DynamoDb.Converters
+{
+    /// <summary>
+    /// Converter for DateTime objects because the default handling expects the data time string to alays be in a very specific
+    /// format and will throw an exception if not.
+    /// </summary>
+    public class DynamoDbDateTimeConverter : IPropertyConverter
+    {
+        public static readonly string DATEFORMAT = "yyyy-MM-ddTHH\\:mm\\:ss.fffffffZ";
+
+        public DynamoDBEntry ToEntry(object value)
+        {
+            if (null == value) return new DynamoDBNull();
+
+            return new Primitive { Value = ((DateTime)value).ToUniversalTime().ToString(DATEFORMAT) };
```

Comment:
I would suggest to remove `ToUniversalTime()` as when I was doing the contact details gateway tests it causes an hour difference between the database and the response object and this was my resolution

Reply:
No. It should be there because we are storing data times as UTC.